packages/shared/routes/setup/legal/Content.svelte

```diff
@@ -101,9 +101,9 @@
 
     <Text type="h2" classes="mt-8 mb-5">2. Data Protection</Text>
     <Text type="p" secondary classes="mb-4">
-        IOTA does not collect any of your personal data through the App. Please refer to the Firefly Privacy Policy (
-        <Link onClick={() => Platform.openUrl('https://firefly.iota.org/privacy')}>
-            https://firefly.iota.org/privacy
+        IOTA does not collect any of your personal data through the App. Please refer to the IOTA Privacy Policy (
```

Comment:
`IOTA Legacy Migration Tool Privacy Policy`

Reply:
since we have this [issue](https://github.com/iotaledger/legacy-migration-tool/issues/6) to update legal content, I did not modify it :see_no_evil: